documentation/docs/tutorials/setup.md

```diff
@@ -528,3 +528,142 @@ The Grafana dashboard should be accessible at `http://<your-ip>:3000`.
+### Software Requirements
+Upgrade your systems' packages by running the following command:
+```bash
+apt update && apt dist-upgrade -y
+```  
+
+#### Installing RocksDB Compression Libraries
+
+GoShimmer uses RocksDB as its underlying database engine. You can install the dependencies required by RocksDB by running the following command:
+
+```bash
+sudo apt-get install libgflags-dev libsnappy-dev zlib1g-dev libbz2-dev liblz4-dev libzstd-dev
+```  
+
+
+####  GCC and G++
+
+GCC and G++ are required for the compilation to work properly.  You can install them by running the following command:
+
+```bash
+sudo apt install gcc g++
+```
+
+### Clone the Repository
+
+Once you have installed the [software requirements](#software-requirements), you should clone the [GoShimmer repository](https://github.com/iotaledger/goshimmer/) into the `/opt` directory. You can do so by running the following commands: 
+
+```bash
+cd /opt
+git clone https://github.com/iotaledger/goshimmer.git
+```
+
+###  Golang-go 1.16
+
+Your system will need to have [Go 1.16](https://go.dev/doc/go1.16) installed for the build script to work. You can install it by running the following command:
+
+```bash
+apt install golang-go
+````
+
+You can make sure you are using the right go version by running the following command:
+
+```bash
+go version
+```
+
+If apt did not install Go version 1.16+, you can use the following commands to download and install it from source:
+
+
+```bash
+cd /home
+wget https://dl.google.com/go/go1.16.4.linux-amd64.tar.gz
+sudo tar -xvf go1.16.4.linux-amd64.tar.gz
+sudo mv go /usr/local
+export GOROOT=/usr/local/go
+cd /opt/goshimmer
+mkdir gopath
+export GOPATH=/opt/goshimmer/gopath
+export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+```
+
+You run `go version` again to check if you have successfully installed `golang-go 1.16`.
+This installation is only valid for this session. If you enter a new session, you will have to enter these commands again:
+
+```bash
+export GOROOT=/usr/local/go
+export GOPATH=/opt/goshimmer/gopath
+export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+```
+
+
+### Download the Snapshot
+You can download the latest snapshot by running the following command from the goshimmer directory you created when you [cloned the repository](#clone-the-repository):
+
+
+```bash
+sudo wget -O snapshot.bin https://dbfiles-goshimmer.s3.eu-central-1.amazonaws.com/snapshots/nectar/snapshot-latest.bin
+```
+
+### Making the Node Dashboard Accessible
+
+You will need to modify your goshimmer configuration file to make the Node Dashboard accessible. 
+1. Make a copy of the [config.default.json](https://github.com/iotaledger/goshimmer/blob/develop/config.default.json) file and rename it config.json.
+2. Using your text editor of choice, edit line 23 to change the `bindAddress` to `0.0.0.0:8081`.
+3. Save your changes.
+
+```bash
+nano config.default.json
+```
+
+On line `23` of the file, change the **bindAddress** to `0.0.0.0:8081`.
```

Comment:
```suggestion
```

Reply:
If the config file structure changes this might not be line 23 anymore. Just say change wherever it says "bindAddress".

Reply:
Done 👍